README.md

```diff
@@ -48,6 +54,68 @@ To use the builder you need to add a new entry to your `architect` object inside
 }
 ```
 
+## `conversionType: files`
+
+- **Builder**: `"@angular-extensions/svg-icons-builder:svg-icons-files-builder"`
+- **Config**: [svg-to-ts files config](https://github.com/kreuzerk/svg-to-ts#3-tree-shakable-and-optimized-for-lazy-loading-conversiontypefiles)
+
+```json
+  "generate-icons": {
+    "builder": "@angular-extensions/svg-icons-builder:svg-icons-files-builder",
+    "options": {
+      "conversionType": "files",
```

Comment:
Do we still need to specify the `conversionType`? I think this is already specified with the builder.